test/config.json

```diff
@@ -791,7 +791,7 @@
 			"Args": ["vitess.io/vitess/go/test/endtoend/topotest/zk2", "--topo-flavor=zk2"],
 			"Command": [],
 			"Manual": false,
-			"Shard": "25",
+			"Shard": "vtgate_topo_sk2",
```

Comment:
tiny typo: sk2 -> zk2

Reply:
Also seems like there should be a workflow file for this?

Reply:
sorry, I did not intend to do this change, I will revert it

Reply:
@deepthi, actually `Cluster 25` does not exist, is this normal?